rtpsender.go

```diff
@@ -175,14 +181,15 @@ func (r *RTPSender) ReplaceTrack(track TrackLocal) error {
 }
 
 // Send Attempts to set the parameters controlling the sending of media.
-func (r *RTPSender) Send(parameters RTPSendParameters) error {
+func (r *RTPSender) Send() error {
```

Comment:
We can't change this API unfortunately. `RTPSender.Send` comes from the [ORTC](https://draft.ortc.org/#dom-rtcrtpsender-send) spec.

In peerconnection.go can we pull the headerExtension values and call `Send` with them?

Reply:
Copy that, I resubmitted it.